examples/yacat/yacat.py

```diff
@@ -67,8 +67,9 @@ async def worker_find_password(ctx: WorkContext, tasks):
 
             # Commands to be run on the provider
             commands = (
+                "rm /golem/work/hashcat.potfile || true; "
```

Comment:
I'm not sure why the change didn't make it to this PR, but removing this file is not enough. `~/.hashcat/hashcat.potfile` should be removed.